packages/builder/src/polywrap_client_config.rs

```diff
@@ -1,48 +1,72 @@
 use std::{collections::HashMap, sync::Arc};
 
 use polywrap_core::{
-    client::{ClientConfig, UriRedirect},
+    client::{ClientConfig, ClientConfigBuilder},
+    interface_implementation::InterfaceImplementations,
     package::WrapPackage,
     resolution::uri_resolver::UriResolver,
     uri::Uri,
     wrapper::Wrapper,
 };
+use polywrap_resolvers::static_resolver::{StaticResolver, StaticResolverLike};
+
+use crate::{PolywrapBaseResolver, PolywrapBaseResolverOptions, PolywrapClientConfigBuilder};
+
+#[derive(Default, Clone)]
+pub struct PolywrapClientConfig {
+    pub interfaces: Option<InterfaceImplementations>,
+    pub envs: Option<HashMap<Uri, Vec<u8>>>,
+    pub wrappers: Option<Vec<(Uri, Arc<dyn Wrapper>)>>,
```

Comment:
NIT: we should start using "wrap" more in our codebases, instead of "wrappers"

Reply:
Good point, I'll do it in a follow up PR